frontend/src/components/table/Table.tsx

```diff
@@ -0,0 +1,28 @@
+import { Table as ATable } from 'antd';
+import styled from 'styled-components';
+
+import { Colors } from '../../styles';
+
+const Table = styled(ATable)`
+    box-shadow: 0px 5px 30px -6px rgba(100, 100, 100, 0.5);
+    overflow-x: auto;
+    background: ${Colors.onPrimary};
+
+    .ant-table-thead > tr > th {
+        font-weight: 400;
+        color: rgba(0, 0, 0, 0.6);
```

Comment:
Ten kolor powinien być wydzielony do stałej `Colors`. I tam wrzuć to jako standardowy kolor bez opacity (tak jak reszta), samo opacity powoduje w tym przypadku, że napisy nie są czarne, tylko takie szarawe, a to jesteś w stanie zapisać w postaci zwykłego hex'a